lib/macos.js

```diff
@@ -45,6 +48,7 @@ var MacosBuilder = {
     options.log( 'Wrote temporary ´appdmg.json´' );
     options.log( 'Kicking off ´appdmg´' );
 
+    var appdmg = require( 'appdmg' );
     var ee = appdmg( {
```

Comment:
Yeah the assert is just stating that you can only attempt to build an osx installer when actually run from osx, it won't even get to the require below. Actually, now that I am more familiar with the code I should probably change that to: `if (process.platform != 'darwin') return callback(new Error('Invalid platform'));`

The assert will just throw and it is probably not very graceful.